src/components/Event.js

```diff
@@ -75,55 +71,6 @@ class Event extends React.Component {
     });
   }
 
-  async removeInterested() {
```

Comment:
This was  already removed by me in another PR, please rebase your branch `git pull --rebase origin master`.

Reply:
@znarf, it shows that my  branch is up to date with origin master. I should rebase it to which commit?

Reply:
@znarf, I am not sure about rebasing it to which particular commit. Running git pull --rebase origin master gives that the branch is up to date with master.

Reply:
Do In need to revert back the commits, rebase it to the master branch and then commit again?

Reply:
@yellowwoods12 looks like the change is gone, so nothing to do anymore